integration-cli/docker_cli_exec_test.go

```diff
@@ -308,7 +308,7 @@ func (s *DockerSuite) TestInspectExecID(c *check.C) {
 	out, _ := dockerCmd(c, "run", "-d", "busybox", "top")
 	id := strings.TrimSuffix(out, "\n")
 
-	out, err := inspectField(id, "ExecIDs")
+	out, err := inspectField(id, ".ExecIDs")
```

Comment:
Hmmm. Not convinced there. I tried this without as it originally was and it fails. I needed to add the dot, same as @duglin did in his linked PR. I'm rebased off master from this afternoon.